src/main/java/com/rocketden/main/exception/RoomErrors.java

```diff
@@ -0,0 +1,16 @@
+package com.rocketden.main.exception;
+
+import com.rocketden.main.exception.api.ApiError;
+import org.springframework.http.HttpStatus;
+
+public class RoomErrors {
```

Comment:
What do you think about combining the RoomErrors and UserErrors? Right now we only have these two types of errors but in the future there could be more categories, and I think it would be more efficient to keep all of them together as a "catalogue" of sorts.

Reply:
Hm, I had the same thought but different conclusion: as we get more types of errors, I was thinking it'd be helpful to have these separate classes to better distinguish errors (e.g. `RoomErrors.NOT_FOUND` and `UserErrors.NOT_FOUND`) and make it easier to refactor if we ever want to split/change the codebase. However, I can also be flexible on this, especially since right now we only have a few errors so it's not too hard to change. @zpChris any thoughts?

Reply:
I think that separate classes is preferable; it let's us easily know which error is attributable to which class, and enables more modularization in the future.

Note: I tried to find if Spectrum's GitHub had any examples to base this decision off of, but I didn't find anything. If you search "exception" on [this Spring GitHub project](https://github.com/spring-projects/greenhouse/find/master), they use separate classes (I know exception and error are different though, so my if you think this is irrelevant that's fine).